packages/fluentui/react-icons-northstar/src/components/DeviceAudioIcon.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+import { createSvgIcon } from '../utils/createSvgIcon';
+
+export const DeviceAudioIcon = createSvgIcon({
```

Comment:
I don't think that it's best name for this but I don't have ideas...

![image](https://user-images.githubusercontent.com/14183168/87543727-7f8be700-c6a5-11ea-808f-95f822803274.png)